README.md

```diff
@@ -549,13 +549,13 @@ author:
   links:
     - title: Twitter
       url: https://twitter.com/username
-      icon: fab fa-twitter-square
+      icon: fa-brands fa-square-twitter
```

Comment:
It felt like updating all the social examples from the base twitter to either x-twitter or bsky would probably be out of scope for this set of changes. Happy to apply across the board if you have a preference.